README.md

```diff
@@ -112,6 +112,7 @@ If you are an open source project maintainer, add the label `first-timers-only`
 - [Pybee/voc](https://github.com/pybee/voc/labels/first-timers-only) _(label: first-timers-only)_ <br> A transpiler that converts Python code into Java bytecode.
 - [Pybee/batavia](https://github.com/pybee/batavia/labels/first-timers-only) _(label: first-timers-only)_ <br> A Javascript implementation of the Python virtual machine.
 - [JARVIS-on-Messenger](https://github.com/swapagarwal/JARVIS-on-Messenger/labels/Low-Hanging%20Fruit) _(label: Low-Hanging Fruit)_ <br> :speech_balloon: A community-driven python bot that aims to be as simple as possible to serve humans with their everyday tasks http://m.me/J.A.R.V.I.S.on.Messenger
+- [Gratipay](https://github.com/gratipay/gratipay.com) _(label: Low-hanging Fruit)_ <br> Helping companies and others pay for open source.
```

Comment:
Would you mind providing a full link to the label like all other repositories in the list?
Thanks!